index.html

```diff
@@ -74,6 +74,11 @@ <h2 class="text-primary">Our Vision</h2>
         </div>
       </div>
     </section>
+    <!-- Buttons  -->
+    <div class="but-section">
+      <button class="btn-primary">Join the Community</button>
+      <button class="btn-secondary"><i class="fas fa-code-branch"></i> Create Pull Requests </button>
```

Comment:
Probably this one :D https://github.com/chryz-hub/chryz-hub.github.io/pulls

Reply:
Okay will do that

Reply:
Added the links